Tests/Unit/Content/StructureResolverTest.php

```diff
@@ -433,6 +477,14 @@ public function testResolvePropertiesIncludeExtension(): void
         $this->contentResolver->resolve('test-123', $titleProperty->reveal(), 'en', ['webspaceKey' => 'sulu_io'])
             ->willReturn($contentView1->reveal());
 
+        $this->referenceStorePool->getStore('content')
```

Comment:
can we use another type (eg. `snippet(s)`) in one of the test cases? 🤔

Reply:
added!